docs/Manifest.toml

```diff
@@ -2,7 +2,7 @@
 
 julia_version = "1.9.1"
```

Comment:
does this mean that if someone is not using 1.9.1, say, 1.9.0, they need to always update the Manifest to agree with what you'd get using julia 1.9.0?  and so every time someone makes a PR with a different version of Julia the Manifest will change?

this seems kind of frustrating to all developers if they arent using the same Julia version.

Reply:
Hmm I'm not exactly sure how this works. I just switched to Julia v1.9.0. I was able to precompile everything, but it does give a warning that recommends to resolve the manifest. So I think you're right that in practice this probably means the Manifest Julia version Manifest will change anytime developers with different versions make a PR. I'm not sure what a good way around this is though, since the Manifest has to have some Julia version specified. Maybe we could all try to stay up to date with the latest version?

Reply:
Discussed offline - Kat and I decided that a good approach may be to create a new release of ClimaLSM using the latest Julia version each time a Julia release is made. We should also make sure that the same julia version is used across Manifests